### ISSUE TITLE
[Support]: Constantly triggering on parked car

### ISSUE THREAD

**Issue by scottjjr** [blakeblackshear/frigate#9001]
### Describe the problem you are having

Hi, have recently upgraded to 13beta6 and yolo7-tiny-416 (was using 12 and either yolo3/4 tiny-416 before), and am  now having a crazy number of 'car entered & left' events for a car that just sits parked in the driveway.  This was an issue on 12 as well but nowhere to the extent i'm getting now.

Here is a screenshot of a couple of hours of recordings (from HA card).

<img width="1687" alt="image" src="https://github.com/blakeblackshear/frigate/assets/57744283/30d236d9-5e18-4305-b22f-e50d93819bac">

Looking at the events, the bounding box changes ever so slightly between detected/left.  Each follows the pattern below, although some (not all) will  become stationary before 'leaving'.

Car detected (even though it was sitting there the whole time)
<img width="959" alt="image" src="https://github.com/blakeblackshear/frigate/assets/57744283/3130faba-cc33-45ca-ba69-f4dc5ef42275">

Car entered Driveway (zone covering my driveway - once again car wasn't moving)
<img width="961" alt="image" src="https://github.com/blakeblackshear/frigate/assets/57744283/0ab023e8-a645-4d7a-94e1-22512db89798">

Car left (car didn't go anywhere )
<img width="955" alt="image" src="https://github.com/blakeblackshear/frigate/assets/57744283/229774e1-858b-42a4-bdfd-3ae803a20d30">

I suspect it's caused by the reflection of clouds moving across the bonnet/windscreen, but I don't understand why it thinks the car is moving/leaving and the bounding boxes are changing slightly.

Occasionally I get a portion of the car determined to be a car, but am more worried about the appearing/disappearing of a parked car triggering events as it fills up with noise and causes a bunch of issues..

Any ideas on what I can tune to minimise these?  The match percentages are high so not even sure I can filter them out., is there a sensitivity to perceived movement setting somewhere I haven't seen?

NB: have trimmed log output to remove base64 images

Thanks,

Scott

### Version

0.13.0-614A36A

### Frigate config file

```yaml
logger:
  default: debug

mqtt:
  host: mqtt.home

detectors:
  tensorrt:
    type: tensorrt

model:
  path: /config/model_cache/tensorrt/yolov7-tiny-416.trt
  input_tensor: nchw
  input_pixel_format: rgb
  width: 416
  height: 416



motion:
  improve_contrast: true

objects:
  track:
  - person
  - bicycle
  - car
  - motorcycle
  - cat
  - dog

ffmpeg:
  hwaccel_args: preset-nvidia-h264

  output_args:
    record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime
      1 -c:v copy -c:a copy

go2rtc:
  streams:
    garage:
    - rtsp://<camera>:554/Streaming/Channels/102
    - ffmpeg:garage#audio=opus

record:
  enabled: true
  retain:
    days: 21
    mode: all
  events:
    retain:
      default: 21
      mode: all

cameras:
  garage:
    ffmpeg:
      inputs:
      - path: rtsp://<camera>:554/Streaming/Channels/102
        roles:
        - detect

      - path: rtsp://<camera>:554/Streaming/Channels/101
        roles:
        - record

    detect:
      width: 1280
      height: 720
      fps: 10

    zones:
      driveway:
        coordinates: 1280,720,1280,311,725,275,390,174,339,220,156,304,66,383,0,271,0,720

      front_lawn:
        coordinates: 157,306,321,187,361,147,292,124,141,102,0,191,0,253,63,395
        filters:
          person:
            min_score: 0.4
            threshold: 0.5

      front_intrusion:
        coordinates: 0,720,510,720,156,303,70,374,0,239

      shared_driveway:
        coordinates: 1280,0,1280,321,853,288,372,178,324,138,134,102,0,196,0,84,59,0,346,0

        filters:
          person:
            min_score: 0.4
            threshold: 0.5
          car:
            min_score: 0.4
            threshold: 0.5
```

### Relevant log output

```shell
2023-12-18 19:33:05.132409009  [2023-12-18 19:33:05] asyncio                        DEBUG   : Using selector: EpollSelector
2023-12-18 19:33:07.695842978  [2023-12-18 19:33:07] asyncio                        DEBUG   : Using selector: EpollSelector
2023-12-18 19:33:07.840673370  [2023-12-18 19:33:07] frigate.events.maintainer      DEBUG   : Event received: tracked_object end garage 1702881162.853018-eisxb7
2023-12-18 19:33:07.840817152  [2023-12-18 19:33:07] peewee.sqliteq                 DEBUG   : received query INSERT INTO "event" ("id", "label", "camera", "start_time", "end_time", "zones", "thumbnail", "has_clip", "has_snapshot", "retain_indefinitely", "ratio", "model_hash", "detector_type", "model_type", "data") VALUES (?, ?, ?, ?, ?, json(?), ?, ?, ?, ?, ?, ?, ?, ?, json(?)) ON CONFLICT ("id") DO UPDATE SET "id" = ?, "label" = ?, "camera" = ?, "start_time" = ?, "end_time" = ?, "zones" = json(?), "thumbnail" = ?, "has_clip" = ?, "has_snapshot" = ?, "model_hash" = ?, "model_type" = ?, "detector_type" = ?, "data" = json(?)
2023-12-18 19:33:07.841075380  [2023-12-18 19:33:07] peewee                         DEBUG   : ('INSERT INTO "event" ("id", "label", "camera", "start_time", "end_time", "zones", "thumbnail", "has_clip", "has_snapshot", "retain_indefinitely", "ratio", "model_hash", "detector_type", "model_type", "data") VALUES (?, ?, ?, ?, ?, json(?), ?, ?, ?, ?, ?, ?, ?, ?, json(?)) ON CONFLICT ("id") DO UPDATE SET "id" = ?, "label" = ?, "camera" = ?, "start_time" = ?, "end_time" = ?, "zones" = json(?), "thumbnail" = ?, "has_clip" = ?, "has_snapshot" = ?, "model_hash" = ?, "model_type" = ?, "detector_type" = ?, "data" = json(?)', ['1702881162.853018-eisxb7', 'car', 'garage', 1702881157.853018, 1702881192.64433, '["driveway"]', '', True, True, '228674ccb92b6d5592f1f089fbd52e9e', <ModelTypeEnum.ssd: 'ssd'>, 'tensorrt', '{"box": [0.39453125, 0.2388888888888889, 0.44765625, 0.7597222222222222], "region": [0.3453125, 0.03333333333333333, 0.54375, 0.9666666666666667], "score": 0.8667389750480652, "top_score": 0.834929883480072, "attributes": [], "type": "object"}'])
2023-12-18 19:33:07.842563047  [2023-12-18 19:33:07] peewee.sqliteq                 DEBUG   : received query INSERT INTO "timeline" ("timestamp", "camera", "source", "source_id", "class_type", "data") VALUES (?, ?, ?, ?, ?, json(?))
2023-12-18 19:33:07.842571386  [2023-12-18 19:33:07] peewee                         DEBUG   : ('INSERT INTO "timeline" ("timestamp", "camera", "source", "source_id", "class_type", "data") VALUES (?, ?, ?, ?, ?, json(?))', [1702881182.470002, 'garage', 'tracked_object', '1702881162.853018-eisxb7', 'gone', '{"box": [0.39453125, 0.24444444444444444, 0.45, 0.7513888888888889], "label": "car", "region": [0.353125, 0.05, 0.534375, 0.95], "attribute": ""}'])
2023-12-18 19:33:07.968002664  [2023-12-18 19:33:07] frigate.comms.ws               DEBUG   : payload for garage/car/snapshot wasn't text. Skipping...
2023-12-18 19:33:08.363046678  [2023-12-18 19:33:08] frigate.events.maintainer      DEBUG   : Event received: tracked_object start garage 1702881188.274838-l2857w
2023-12-18 19:33:08.363861034  [2023-12-18 19:33:08] peewee.sqliteq                 DEBUG   : received query INSERT INTO "timeline" ("timestamp", "camera", "source", "source_id", "class_type", "data") VALUES (?, ?, ?, ?, ?, json(?))
2023-12-18 19:33:08.363949556  [2023-12-18 19:33:08] peewee                         DEBUG   : ('INSERT INTO "timeline" ("timestamp", "camera", "source", "source_id", "class_type", "data") VALUES (?, ?, ?, ?, ?, json(?))', [1702881188.274838, 'garage', 'tracked_object', '1702881188.274838-l2857w', 'visible', '{"box": [0.396875, 0.24166666666666667, 0.4390625, 0.7569444444444444], "label": "car", "region": [0.353125, 0.06111111111111111, 0.528125, 0.9388888888888889], "attribute": ""}'])
2023-12-18 19:33:08.471193758  [2023-12-18 19:33:08] frigate.events.maintainer      DEBUG   : Event received: tracked_object update garage 1702881188.274838-l2857w
2023-12-18 19:33:08.471352428  [2023-12-18 19:33:08] peewee.sqliteq                 DEBUG   : received query INSERT INTO "event" ("id", "label", "camera", "start_time", "end_time", "zones", "thumbnail", "has_clip", "has_snapshot", "retain_indefinitely", "ratio", "model_hash", "detector_type", "model_type", "data") VALUES (?, ?, ?, ?, ?, json(?), ?, ?, ?, ?, ?, ?, ?, ?, json(?)) ON CONFLICT ("id") DO UPDATE SET "id" = ?, "label" = ?, "camera" = ?, "start_time" = ?, "end_time" = ?, "zones" = json(?), "thumbnail" = ?, "has_clip" = ?, "has_snapshot" = ?, "model_hash" = ?, "model_type" = ?, "detector_type" = ?, "data" = json(?)
2023-12-18 19:33:08.471581626  [2023-12-18 19:33:08] peewee                         DEBUG   : ('INSERT INTO "event" ("id", "label", "camera", "start_time", "end_time", "zones", "thumbnail", "has_clip", "has_snapshot", "retain_indefinitely", "ratio", "model_hash", "detector_type", "model_type", "data") VALUES (?, ?, ?, ?, ?, json(?), ?, ?, ?, ?, ?, ?, ?, ?, json(?)) ON CONFLICT ("id") DO UPDATE SET "id" = ?, "label" = ?, "camera" = ?, "start_time" = ?, "end_time" = ?, "zones" = json(?), "thumbnail" = ?, "has_clip" = ?, "has_snapshot" = ?, "model_hash" = ?, "model_type" = ?, "detector_type" = ?, "data" = json(?)', ['1702881188.274838-l2857w', 'car', 'garage', 1702881183.274838, None, '[]', '', True, True, False, 1.0, '228674ccb92b6d5592f1f089fbd52e9e', 'tensorrt', <ModelTypeEnum.ssd: 'ssd'>, '{"box": [0.3953125, 0.22361111111111112, 0.45, 0.775], "region": [0.3421875, 0.03333333333333333, 0.54375, 0.9666666666666667], "score": 0.7039588093757629, "top_score": 0.7053658962249756, "attributes": [], "type": "object"}', '1702881188.274838-l2857w', 'car', 'garage', 1702881183.274838, None, '[]', '', True, True, '228674ccb92b6d5592f1f089fbd52e9e', <ModelTypeEnum.ssd: 'ssd'>, 'tensorrt', '{"box": [0.3953125, 0.22361111111111112, 0.45, 0.775], "region": [0.3421875, 0.03333333333333333, 0.54375, 0.9666666666666667], "score": 0.7039588093757629, "top_score": 0.7053658962249756, "attributes": [], "type": "object"}'])
2023-12-18 19:33:08.767008229  [2023-12-18 19:33:08] frigate.events.maintainer      DEBUG   : Event received: tracked_object update garage 1702881188.274838-l2857w
2023-12-18 19:33:08.767159207  [2023-12-18 19:33:08] peewee.sqliteq                 DEBUG   : received query INSERT INTO "event" ("id", "label", "camera", "start_time", "end_time", "zones", "thumbnail", "has_clip", "has_snapshot", "retain_indefinitely", "ratio", "model_hash", "detector_type", "model_type", "data") VALUES (?, ?, ?, ?, ?, json(?), ?, ?, ?, ?, ?, ?, ?, ?, json(?)) ON CONFLICT ("id") DO UPDATE SET "id" = ?, "label" = ?, "camera" = ?, "start_time" = ?, "end_time" = ?, "zones" = json(?), "thumbnail" = ?, "has_clip" = ?, "has_snapshot" = ?, "model_hash" = ?, "model_type" = ?, "detector_type" = ?, "data" = json(?)
2023-12-18 19:33:08.767434786  [2023-12-18 19:33:08] peewee                         DEBUG   : ('INSERT INTO "event" ("id", "label", "camera", "start_time", "end_time", "zones", "thumbnail", "has_clip", "has_snapshot", "retain_indefinitely", "ratio", "model_hash", "detector_type", "model_type", "data") VALUES (?, ?, ?, ?, ?, json(?), ?, ?, ?, ?, ?, ?, ?, ?, json(?)) ON CONFLICT ("id") DO UPDATE SET "id" = ?, "label" = ?, "camera" = ?, "start_time" = ?, "end_time" = ?, "zones" = json(?), "thumbnail" = ?, "has_clip" = ?, "has_snapshot" = ?, "model_hash" = ?, "model_type" = ?, "detector_type" = ?, "data" = json(?)', ['1702881188.274838-l2857w', 'car', 'garage', 1702881183.274838, None, '["driveway"]', '', True, True, False, 1.0, '228674ccb92b6d5592f1f089fbd52e9e', 'tensorrt', <ModelTypeEnum.ssd: 'ssd'>, '{"box": [0.3953125, 0.22361111111111112, 0.45, 0.775], "region": [0.3421875, 0.03333333333333333, 0.54375, 0.9666666666666667], "score": 0.7039588093757629, "top_score": 0.7053658962249756, "attributes": [], "type": "object"}', '1702881188.274838-l2857w', 'car', 'garage', 1702881183.274838, None, '["driveway"]', '', True, True, '228674ccb92b6d5592f1f089fbd52e9e', <ModelTypeEnum.ssd: 'ssd'>, 'tensorrt', '{"box": [0.3953125, 0.22361111111111112, 0.45, 0.775], "region": [0.3421875, 0.03333333333333333, 0.54375, 0.9666666666666667], "score": 0.7039588093757629, "top_score": 0.7053658962249756, "attributes": [], "type": "object"}'])
2023-12-18 19:33:08.768942278  [2023-12-18 19:33:08] peewee.sqliteq                 DEBUG   : received query INSERT INTO "timeline" ("timestamp", "camera", "source", "source_id", "class_type", "data") VALUES (?, ?, ?, ?, ?, json(?))
2023-12-18 19:33:08.769040250  [2023-12-18 19:33:08] peewee                         DEBUG   : ('INSERT INTO "timeline" ("timestamp", "camera", "source", "source_id", "class_type", "data") VALUES (?, ?, ?, ?, ?, json(?))', [1702881188.644747, 'garage', 'tracked_object', '1702881188.274838-l2857w', 'entered_zone', '{"box": [0.3953125, 0.23055555555555557, 0.44921875, 0.7680555555555556], "label": "car", "region": [0.34296875, 0.022222222222222223, 0.55, 0.9777777777777777], "attribute": "", "zones": ["driveway"]}'])
2023-12-18 19:33:10.132291889  [2023-12-18 19:33:10] asyncio                        DEBUG   : Using selector: EpollSelector
2023-12-18 19:33:10.796877903  [2023-12-18 19:33:10] frigate.record.maintainer      DEBUG   : Copied /media/frigate/recordings/2023-12-18/06/garage/32.55.mp4 in 0.3981130123138428 seconds.
2023-12-18 19:33:10.801619106  [2023-12-18 19:33:10] peewee.sqliteq                 DEBUG   : received query INSERT INTO "recordings" ("id", "camera", "path", "start_time", "end_time", "duration", "motion", "objects", "dBFS", "segment_size") VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
2023-12-18 19:33:10.801763500  [2023-12-18 19:33:10] peewee                         DEBUG   : ('INSERT INTO "recordings" ("id", "camera", "path", "start_time", "end_time", "duration", "motion", "objects", "dBFS", "segment_size") VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)', ['1702881175.0-g7vkim', 'garage', '/media/frigate/recordings/2023-12-18/06/garage/32.55.mp4', 1702881175.0, 1702881184.969336, 9.969335937499999, 832611, 0, 0, 18.9])
2023-12-18 19:33:11.936984088  [2023-12-18 19:33:11] frigate.events.maintainer      DEBUG   : Event received: tracked_object end garage 1702881188.274838-l2857w
2023-12-18 19:33:11.937215896  [2023-12-18 19:33:11] peewee.sqliteq                 DEBUG   : received query INSERT INTO "event" ("id", "label", "camera", "start_time", "end_time", "zones", "thumbnail", "has_clip", "has_snapshot", "retain_indefinitely", "ratio", "model_hash", "detector_type", "model_type", "data") VALUES (?, ?, ?, ?, ?, json(?), ?, ?, ?, ?, ?, ?, ?, ?, json(?)) ON CONFLICT ("id") DO UPDATE SET "id" = ?, "label" = ?, "camera" = ?, "start_time" = ?, "end_time" = ?, "zones" = json(?), "thumbnail" = ?, "has_clip" = ?, "has_snapshot" = ?, "model_hash" = ?, "model_type" = ?, "detector_type" = ?, "data" = json(?)
2023-12-18 19:33:11.937392236  [2023-12-18 19:33:11] peewee                         DEBUG   : ('INSERT INTO "event" ("id", "label", "camera", "start_time", "end_time", "zones", "thumbnail", "has_clip", "has_snapshot", "retain_indefinitely", "ratio", "model_hash", "detector_type", "model_type", "data") VALUES (?, ?, ?, ?, ?, json(?), ?, ?, ?, ?, ?, ?, ?, ?, json(?)) ON CONFLICT ("id") DO UPDATE SET "id" = ?, "label" = ?, "camera" = ?, "start_time" = ?, "end_time" = ?, "zones" = json(?), "thumbnail" = ?, "has_clip" = ?, "has_snapshot" = ?, "model_hash" = ?, "model_type" = ?, "detector_type" = ?, "data" = json(?)', ['1702881188.274838-l2857w', 'car', 'garage', 1702881183.274838, 1702881196.745432, '["driveway"]', '', True, True, False, 1.0, '228674ccb92b6d5592f1f089fbd52e9e', 'tensorrt', <ModelTypeEnum.ssd: 'ssd'>, '{"box": [0.39765625, 0.23472222222222222, 0.4515625, 0.7638888888888888], "region": [0.35, 0.05, 0.534375, 0.95], "score": 0.8032331466674805, "top_score": 0.7053658962249756, "attributes": [], "type": "object"}', '1702881188.274838-l2857w', 'car', 'garage', 1702881183.274838, 1702881196.745432, '["driveway"]', '', True, True, '228674ccb92b6d5592f1f089fbd52e9e', <ModelTypeEnum.ssd: 'ssd'>, 'tensorrt', '{"box": [0.39765625, 0.23472222222222222, 0.4515625, 0.7638888888888888], "region": [0.35, 0.05, 0.534375, 0.95], "score": 0.8032331466674805, "top_score": 0.7053658962249756, "attributes": [], "type": "object"}'])
2023-12-18 19:33:11.939172842  [2023-12-18 19:33:11] peewee.sqliteq                 DEBUG   : received query INSERT INTO "timeline" ("timestamp", "camera", "source", "source_id", "class_type", "data") VALUES (?, ?, ?, ?, ?, json(?))
2023-12-18 19:33:11.939178137  [2023-12-18 19:33:11] peewee                         DEBUG   : ('INSERT INTO "timeline" ("timestamp", "camera", "source", "source_id", "class_type", "data") VALUES (?, ?, ?, ?, ?, json(?))', [1702881191.37156, 'garage', 'tracked_object', '1702881188.274838-l2857w', 'gone', '{"box": [0.39296875, 0.2263888888888889, 0.453125, 0.7722222222222223], "label": "car", "region": [0.353125, 0.03333333333333333, 0.54375, 0.9666666666666667], "attribute": ""}'])
2023-12-18 19:33:12.067060189  [2023-12-18 19:33:12] frigate.comms.ws               DEBUG   : payload for garage/car/snapshot wasn't text. Skipping...
2023-12-18 19:33:12.660131460  [2023-12-18 19:33:12] frigate.events.maintainer      DEBUG   : Event received: tracked_object start garage 1702881192.545574-jgappy
2023-12-18 19:33:12.661105814  [2023-12-18 19:33:12] peewee.sqliteq                 DEBUG   : received query INSERT INTO "timeline" ("timestamp", "camera", "source", "source_id", "class_type", "data") VALUES (?, ?, ?, ?, ?, json(?))
2023-12-18 19:33:12.661200219  [2023-12-18 19:33:12] peewee                         DEBUG   : ('INSERT INTO "timeline" ("timestamp", "camera", "source", "source_id", "class_type", "data") VALUES (?, ?, ?, ?, ?, json(?))', [1702881192.545574, 'garage', 'tracked_object', '1702881192.545574-jgappy', 'visible', '{"box": [0.396875, 0.24166666666666667, 0.44921875, 0.7569444444444444], "label": "car", "region": [0.35390625, 0.05555555555555555, 0.53125, 0.9444444444444444], "attribute": ""}'])
```

### FFprobe output from your camera

```shell
FFPROBE OUTPUT
Stream 0:
Return Code: 0

Video:

Codec: H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10
Resolution: 1280x720
FPS: 10/1

Audio:

Codec: AAC (Advanced Audio Coding)

Stream 1:
Return Code: 0

Video:

Codec: H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10
Resolution: 3840x2160
FPS: 20/1

Audio:

Codec: AAC (Advanced Audio Coding)

COPY
CLOSE
```

### Frigate stats

```json
{
    "cameras": {
        "garage": {
            "audio_dBFS": 0,
            "audio_rms": 0,
            "camera_fps": 10,
            "capture_pid": 336,
            "detection_enabled": 1,
            "detection_fps": 10,
            "ffmpeg_pid": 340,
            "pid": 334,
            "process_fps": 10,
            "skipped_fps": 0
        }
    },
    "cpu_usages": {
        "1": {
            "cmdline": "/package/admin/s6/command/s6-svscan -d4 -- /run/service",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "15": {
            "cmdline": "s6-supervise s6-linux-init-shutdownd",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "17": {
            "cmdline": "/package/admin/s6-linux-init/command/s6-linux-init-shutdownd -c /run/s6/basedir -g 3000 -C -B",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "24": {
            "cmdline": "s6-supervise s6rc-oneshot-runner",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "25": {
            "cmdline": "s6-supervise s6rc-fdholder",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "26": {
            "cmdline": "s6-supervise frigate",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "27": {
            "cmdline": "s6-supervise frigate-log",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "28": {
            "cmdline": "s6-supervise go2rtc",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "29": {
            "cmdline": "s6-supervise go2rtc-healthcheck",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "30": {
            "cmdline": "s6-supervise go2rtc-log",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "31": {
            "cmdline": "s6-supervise nginx",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "32": {
            "cmdline": "s6-supervise nginx-log",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "43": {
            "cmdline": "/package/admin/s6-2.11.3.2/command/s6-fdholderd -1 -i data/rules",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "44": {
            "cmdline": "/package/admin/s6/command/s6-ipcserverd -1 -- /package/admin/s6/command/s6-ipcserver-access -v0 -E -l0 -i data/rules -- /package/admin/s6/command/s6-sudod -t 30000 -- /package/admin/s6-rc/command/s6-rc-oneshot-run -l ../.. --",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "87": {
            "cmdline": "s6-log -b -- T 1 n0 s10000000 T /dev/shm/logs/nginx",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "89": {
            "cmdline": "s6-log -b -- T 1 n0 s10000000 T /dev/shm/logs/frigate",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "92": {
            "cmdline": "s6-log -b -- T 1 n0 s10000000 T /dev/shm/logs/go2rtc",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "98": {
            "cmdline": "/usr/local/go2rtc/bin/go2rtc -config=/dev/shm/go2rtc.yaml",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.3"
        },
        "117": {
            "cmdline": "bash ./run.user go2rtc-healthcheck",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "118": {
            "cmdline": "python3 -u -m frigate",
            "cpu": "1.2",
            "cpu_average": "1",
            "mem": "4.3"
        },
        "128": {
            "cmdline": "nginx: master process nginx",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.1"
        },
        "153": {
            "cmdline": "nginx: worker process",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "1.2"
        },
        "154": {
            "cmdline": "nginx: worker process",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "1.1"
        },
        "155": {
            "cmdline": "nginx: worker process",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.9"
        },
        "156": {
            "cmdline": "nginx: worker process",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "1.1"
        },
        "162": {
            "cmdline": "nginx: cache manager process",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        },
        "306": {
            "cmdline": "frigate.logger       ",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "2.0"
        },
        "308": {
            "cmdline": "frigate.recording_manager",
            "cpu": "0.2",
            "cpu_average": "2",
            "mem": "3.3"
        },
        "317": {
            "cmdline": "/usr/bin/python3 -c from multiprocessing.resource_tracker import main;main(41)",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.1"
        },
        "319": {
            "cmdline": "frigate.detector.tensorrt",
            "cpu": "9.3",
            "cpu_average": "5",
            "mem": "5.4"
        },
        "322": {
            "cmdline": "frigate.output       ",
            "cpu": "1.0",
            "cpu_average": "0",
            "mem": "2.1"
        },
        "326": {
            "cmdline": "ffmpeg -f rawvideo -pix_fmt yuv420p -video_size 1280x720 -i pipe: -f mpegts -s 1280x720 -codec:v mpeg1video -q 8 -bf 0 pipe:",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.1"
        },
        "327": {
            "cmdline": "ffmpeg -f rawvideo -pix_fmt yuv420p -video_size 1280x720 -i pipe: -f mpegts -s 1280x720 -codec:v mpeg1video -q 8 -bf 0 pipe:",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.1"
        },
        "334": {
            "cmdline": "frigate.process:garage",
            "cpu": "6.2",
            "cpu_average": "5",
            "mem": "2.5"
        },
        "336": {
            "cmdline": "frigate.capture:garage",
            "cpu": "2.5",
            "cpu_average": "2",
            "mem": "2.1"
        },
        "340": {
            "cmdline": "ffmpeg -hide_banner -loglevel warning -threads 2 -hwaccel cuda -hwaccel_output_format cuda -user_agent FFmpeg Frigate/0.13.0-614a36a -avoid_negative_ts make_zero -fflags +genpts+discardcorrupt -rtsp_transport tcp -timeout 5000000 -use_wallclock_as_timestamps 1 -i rtsp://admin:lBIwZOh%214%21lVl4kA@10.1.1.9:554/Streaming/Channels/102 -r 10 -vf fps=10,scale_cuda=w=1280:h=720:format=nv12,hwdownload,format=nv12,format=yuv420p -threads 2 -f rawvideo -pix_fmt yuv420p pipe:",
            "cpu": "5.8",
            "cpu_average": "5",
            "mem": "3.8"
        },
        "3574": {
            "cmdline": "ffmpeg -hide_banner -loglevel warning -threads 2 -hwaccel cuda -hwaccel_output_format cuda -user_agent FFmpeg Frigate/0.13.0-614a36a -avoid_negative_ts make_zero -fflags +genpts+discardcorrupt -rtsp_transport tcp -timeout 5000000 -use_wallclock_as_timestamps 1 -i rtsp://admin:lBIwZOh%214%21lVl4kA@10.1.1.9:554/Streaming/Channels/101 -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a copy /tmp/cache/garage@%Y%m%d%H%M%S%z.mp4",
            "cpu": "2.7",
            "cpu_average": "3",
            "mem": "0.2"
        },
        "4175": {
            "cmdline": "sleep 30s",
            "cpu": "0.0",
            "cpu_average": "0",
            "mem": "0.0"
        }
    },
    "detection_fps": 10,
    "detectors": {
        "tensorrt": {
            "detection_start": 0,
            "inference_speed": 9.15,
            "pid": 319
        }
    },
    "gpu_usages": {
        "Quadro P1000": {
            "dec": "1.0%",
            "enc": "0.0%",
            "gpu": "9.0%",
            "mem": "7.71%"
        }
    },
    "processes": {
        "go2rtc": {
            "pid": 98
        },
        "logger": {
            "pid": 306
        },
        "recording": {
            "pid": 308
        }
    },
    "service": {
        "last_updated": 1702890955,
        "latest_version": "0.12.1",
        "storage": {
            "/dev/shm": {
                "free": 248.3,
                "mount_type": "tmpfs",
                "total": 256,
                "used": 7.7
            },
            "/media/frigate/clips": {
                "free": 5351158.4,
                "mount_type": "fuse.shfs",
                "total": 5721128.5,
                "used": 369970.1
            },
            "/media/frigate/recordings": {
                "free": 5351158.4,
                "mount_type": "fuse.shfs",
                "total": 5721128.5,
                "used": 369970.1
            },
            "/tmp/cache": {
                "free": 180.1,
                "mount_type": "tmpfs",
                "total": 190.7,
                "used": 10.6
            }
        },
        "temperatures": {},
        "uptime": 202336,
        "version": "0.13.0-614a36a"
    }
}
```

### Operating system

unraid

### Install method

docker unraid

### Coral version

None (nvidia)

### Network connection

Wired

### Camera make and model

Hilook IPC-T281H-MU

**Comment by scottjjr** [comment]
Here's one from this evening where the car is detected, then 'leaves' with a different bounding box.  There are no reflection/shadow changes.

Detected
<img width="961" alt="image" src="https://github.com/blakeblackshear/frigate/assets/57744283/03878f94-7493-4418-b236-6d6de8ae121b">

Entered 'driveway'
<img width="962" alt="image" src="https://github.com/blakeblackshear/frigate/assets/57744283/c1b4bbf3-ed96-4448-8b70-57424caa85c5">


Car 'left'
<img width="963" alt="image" src="https://github.com/blakeblackshear/frigate/assets/57744283/50ecc6e3-f39f-451e-a5a3-6918be9248a0">

**Comment by NickM-27** [comment]
motion detection might be too sensitive, the docs cover how to tune motion detection https://deploy-preview-6262--frigate-docs.netlify.app/configuration/motion_detection

**Comment by scottjjr** [comment]
Thanks will give it a go and see if it helps.

**Comment by nwish** [comment]
I have this exact same issue, it’s been persistent through all .13 releases but I definitely believe it’s a matter of tuning the settings.  I managed to greatly reduce the number of events from this scenario by increasing the zone_inertia setting for my “driveway” zone, which tells Frigate that an object must be in that zone for X number of consecutive frames before legitimately being considered IN the zone.  This cut back on the events created when the bounding box gets all silly and jumps into the zone for a few frames and then back out.  I spent a lot of time watching the debug feed lol… so anyway, try setting inertia for that zone to a number of frames equivalent to more than a second and see if that helps.

**Comment by scottjames** [comment]
https://deploy-preview-6262--frigate-docs.netlify.app/configuration/zones#zone-inertia

I also found this helpful for parked cars:
https://deploy-preview-6262--frigate-docs.netlify.app/configuration/stationary_objects

**Comment by autumnwalker** [comment]
@scottjjr - any luck with motion detection tuning? I'm having the same problem that you are describing on 0.13.0-49814B3.

**Comment by scottjjr** [comment]
I ended up winding the motion sensitivity down a lot, which has dropped the number of false movements considerably.  Unsure if I'm missing events I'd be interested in but at least the noise has gone down.  

Most of what I'm interested in is moving across camera (up/down the shared driveway), or close to camera (as people approach the house/enter my driveway off the shared driveway), so tracking only the bigger movements is likely fine.

I'd also recently got a reolink doorbell which looks over the same area from a different angle.  This also suffered from false movements so have applied the same settings to that also.  Settings I used below for reference.

      contour_area: 70
      threshold: 60
